### PR TITLE
CPDP-1896: Fixes for sbt toplecat plugin warnings and errors

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -0,0 +1 @@
+GET        /admin/metrics                                           com.kenshoo.play.metrics.MetricsController.metrics
```
Comment (AndyHWChung) [inline conf/app.routes:1]: shouldn't this stay in `prod.routes` - if it's here then the url will be expected to be prefixed by `/cgt-property-disposals-stubs` now when it wasn't before

File: app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/FinancialDataController.scala [hunk 2]
```diff
@@ -33,6 +33,11 @@ class FinancialDataController @Inject() (cc: ControllerComponents) extends Backe
     dateFrom: String,
     dateTo: String
   ): Action[AnyContent] = Action { _ =>
+    logger.info(s"""
+        idType = $idType, idNumber = $idNumber,
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/FinancialDataController.scala:37]: it'd be handy to have an indication about what endpoint is being called in the log message, e.g. `"Get financial data called with parameters: idType = $idType..."`

File: conf/app.routes (hunk 1 above)
Comment (rambabu-posa) [reply conf/app.routes:1]: Earlier I had some unused import issues with this file. I just moved this to app.routes to fix those issues. Let me test whether its required or not with new configs

Comment (rambabu-posa) [reply conf/app.routes:1]: If I did this change, unfortunately Im getting these error:
`[error] /Users/ram/Applications/hmrc/CGTProjects/cgt-property-disposals-stubs/conf/app.routes: Unused import
[error] /Users/ram/Applications/hmrc/CGTProjects/cgt-property-disposals-stubs/conf/app.routes: private val defaultPrefix in class Routes is never used`

Is this app.routes really required? Its just empty

Comment (AndyHWChung) [reply conf/app.routes:1]: it isn't required for now, but may be for the future. Let's delete it for now if it's not being used then and add it back in if we need it in the future

**Pull request merged**